### PR TITLE
Global Sensitivity Analysis with QUESO

### PR DESCRIPTION
**Pull request by vohramanav**
I have written a new chapter (#5) for the user manual. It provides a theoretical background to Global Sensitivity Analysis and a discussion on how we could exploit the SFP in QUESO to perform prior-based parametric sensitivity analysis. The idea is conveyed using an algorithm to generate the required set of data. A simple application involving sensitivity analysis of the slope and y-intercept of a straight line is presented. Snippets of the actual code are also included for reference purposes. 

Through this initiative, I sincerely hope that the user manual would serve as a reference document, consistent with the most recent release of QUESO. I have proposed archiving it on arXiv.org for future updates and ease of access to the growing user-base.

### CODE REVIEW

**Comment by dmcdougall** [comment]
I can't build the manual:

```
! Package Listings Error: File `/localscratch/eureka/line_inference/twoD/with_S
ensitivity/sensitivity_mc(.h)' not found.
```

Perhaps you forgot to `git add` a file?

**Comment by dmcdougall** [comment]
Thanks for the files.  The LaTeX source still refers to a non-existent file, though.  See [here](https://github.com/libqueso/queso/pull/477/files#diff-90c1f5743279bf7aaa83c5065b25fee9R184).  You'll need to update the path to point to the (relative) path to the file in the `manual` directory.

**Comment by vohramanav** [comment]
Forgot to commit the source file.. should be fine now. Please let me know. Thanks!

**Comment by dmcdougall** [comment]
Absolute paths won't work.

> You'll need to update the path to point to the **(relative)** path to the file in the manual directory.

**Comment by vohramanav** [comment]
I see. Does it work now?

**Comment by dmcdougall** [comment]
Yes.

**Comment by vohramanav** [comment]
Awesome. Sorry about the inconvenience!

File: manual/users/users_gsa.tex [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+\cmidrule(lr){2-2}\cmidrule(lr){3-3}
+& $\frac{1}{2N}\sum_{k=1}^{N} (f(\bm A)_{k} - f(\bm A_{\bm B}^{(i)})_{k})^{2}$ & Jansen 1999~\cite{Jansen:1999}   \\
+\bottomrule
+\end{tabular}
+\caption{Commonly used estimators and corresponding references for $\V_{\theta_{i}}(\E_{\bm \theta_{\sim i}}[\mathcal{G}\vert\theta_{i}])$ and $\E_{\bm \theta_{\sim i}}[\V_{\theta_{i}}(\mathcal{G}\vert\bm{\theta}_{\sim i})]$.}
+\label{tab:estimators}
+\end{table}
+
+Statistical forward problem (SFP) can be solved with QUESO by computing the QoI for pseudo-random samples drawn 
+from the posterior distribution as discussed in~\ref{sub:SFP}. However, for GSA, we need to generate two independent
```
Comment (dmcdougall) [inline manual/users/users_gsa.tex]: `sub:SFP` isn't defined.

File: manual/users/users_gsa.tex [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+set $\bm B$ and corresponding QoI estimates.
+\item \texttt{m\_qoi\_samplesBi.txt}: All columns from set $\bm B$ except the $i^{th}$ ($i$=1) column which is from
+set $\bm A$ and corresponding QoI estimates.
+\item \texttt{c\_qoi\_samplesAi.txt}: All columns from set $\bm A$ except the $i^{th}$ ($i$=2) column which is from
+set $\bm B$ and corresponding QoI estimates.
+\item \texttt{c\_qoi\_samplesBi.txt}: All columns from set $\bm B$ except the $i^{th}$ ($i$=2) column which is from
+set $\bm A$ and corresponding QoI estimates.
+\end{enumerate}
+
+In the above list, let use denote files in 3---6 as the set of derived files.  The following algorithm provides
```
Comment (dmcdougall) [inline manual/users/users_gsa.tex]: The first sentence here doesn't make sense.

File: manual/users/users_gsa.tex [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+\caption*{\textbf{Algorithm}:~Generating data for GSA}
+\begin{algorithmic}[1]
+\Procedure{Solving SFP with QUESO}{}
+\BState Instantiate a QoI object (\texttt{qoi\_mc}): 
+\Statex \texttt{Qoi\_mc<> qoi\_mc("qoi\_", paramDomain, qoiSpace);}
+\BState Instantiate the forward problem (\texttt{fp\_mc}):
+\Statex \texttt{QUESO::StatisticalForwardProblem<> fp\_mc("", NULL, priorRv, qoi\_mc, qoiRv);} 
+\BState Solve the forward problem to generate the data file, \texttt{qoi\_samplesA.txt} i.e. set $\bm A$. 
+\Statex \texttt{fp\_mc.solveWithMonteCarlo(NULL);}
+\BState Repeat steps 2---4 to generate the data file, \texttt{qoi\_samplesB.txt} i.e. set $\bm B$.
```
Comment (dmcdougall) [inline manual/users/users_gsa.tex]: [Use `--` for ranges](http://tex.stackexchange.com/questions/3819/dashes-vs-vs).

Comment (dmcdougall) [inline manual/users/users_gsa.tex]: There are other places where this occurs.

File: manual/users/users_gsa.tex (hunk 1 above)
Comment (vohramanav) [inline manual/users/users_gsa.tex]: I did have the label but did not commit the update. It is fixed now.

File: manual/users/users_gsa.tex (hunk 2 above)
Comment (vohramanav) [inline manual/users/users_gsa.tex]: The sentence has been updated.

File: manual/users/users_gsa.tex (hunk 3 above)
Comment (vohramanav) [inline manual/users/users_gsa.tex]: Fixed.

**Comment by vohramanav** [comment]
Thanks Damon. Did you get a chance to update the cover?

**Pull request closed without merging**